eng/pipelines/templates/jobs/archetype-sdk-client-test.yml

```diff
@@ -0,0 +1,84 @@
+parameters:
```

Comment:
I foresee this template and the archetype-sdk-tests.yml being confused with one another. I already had to look twice t understand the difference. I have to wonder how close the 2 are today and how difficult it would be to parameterize the 2.